### ISSUE TITLE
Not sending Campains

### ISSUE THREAD

**Issue by stuxve** [PaulleDemon/Email-automation#7]
Hi,

I installed the tool and registered successfully. I also added a custom email account to send campaigns. However, when I schedule a campaign, it switches from "Started: Yes" to "Started: No" at the scheduled start time, and the campaign does not proceed as expected.

Could you please assist with this issue?

Thank you!

![image](https://github.com/user-attachments/assets/3817e0ce-974c-4a07-9db6-617ac17d8d25)

**Comment by PaulleDemon** [comment]
Hey, 
I am no longer using or maintaining this. However, can you check your log files?

**Comment by stuxve** [comment]
Oh sad to hear it. The app have potential...

The web application is not generating any logs. But in the output of the commands is not even trying to do anything.

![image](https://github.com/user-attachments/assets/008d83ed-c996-4d29-b6ad-74afd0ac8bb0)

**Comment by PaulleDemon** [comment]
did you run both these commands?

```
python manage.py runserver
```
and
```
celery -A email_automation worker -l info -B --scheduler django_celery_beat.schedulers:DatabaseScheduler 
```

also did you check if emails are being sent? did you configure the emails in the configure email section?

**Comment by stuxve** [comment]
Yes, I have both commands running, and the email configured in the email section. I wanted to check if it works sending, but... no even trying.

**Comment by PaulleDemon** [comment]
Hey, its likely that there was an error sending the message, did you upload excel file with recipient emails in it?

You should have an Email column in the excel file, or it won't be received.

**Comment by stuxve** [comment]
I create a script to create the excel, the import does not throw any error (test it's just random data in this text, in the code is my real email):

```
import pyexcel as pe

data = [["Email"], ["test@gmail.com"]]
pe.save_as(array=data, dest_file_name="email_list.xls")

```
